docs/ops/movement/ScatterElementsUpdate_12.md

```diff
@@ -0,0 +1,209 @@
+# ScatterElementsUpdate {#openvino_docs_ops_movement_ScatterElementsUpdate_12}
+
+@sphinxdirective
+
+**Versioned name**: *ScatterElementsUpdate-12*
+
+**Category**: *Data movement*
+
+**Short description**: Creates a copy of the first input tensor with elements updated according to the logic specified by *reduction* attribute and the following inputs: ``indices``, ``updates``, ``axis``.
+
+**Detailed description**: For each element of ``updates`` input, the value of ``indices`` at the same index points the element to update on the ``axis`` of ``data``. The index for dimension pointed by ``axis`` is provided by values of ``indices`` input, otherwise the index is the same as the index of the entry itself.
+
+The dimensions of ``update`` tensor are allowed to be less or equal to the corresponding dimensions of ``data`` tensor, but the dimension pointed by ``axis`` can be also greater (especially if the ``indices`` input contains duplicated values).
+
+The operation to perform between the corresponding elements is specified by *reduction* attribute,
+by default the elements of ``data`` tensor are simply overwritten by the values from ``updates`` input.
+
+Additionally *use_init_val* attribute can be used to control whether the elements from the ``data`` input tensor are used as initial value (enabled by default).
+
+For instance, in a 3D tensor case, the update of the element corresponding to the ``[i][j][k]`` is performed as below:
+
```

Comment:
may be it makes sense to add concrete examples with 2D input tensors for better explanation of below formulas

Reply:
Added more 2D examples with values in the Examples section.